huaweicloud/resource_huaweicloud_cce_node_attach.go

```diff
@@ -418,10 +422,14 @@ func resourceCCENodeAttachV3Update(ctx context.Context, d *schema.ResourceData,
 				SshKey: d.Get("key_pair").(string),
 			}
 		} else {
+			password, err := utils.TryPasswordEncrypt(d.Get("password").(string))
```

Comment:
If we support password update, need to consider whether it really changed between plain and salted password, suppress the diff.

Reply:
fair enough